testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1281,3 +1281,13 @@ def server_secret
   puts "URL: #{url}"
   step %(I enter "#{url}" as "calendar-data-text")
 end
+
+When(/^I deploy testing playbooks and inventory files to "([^"]*)"$/) do |host|
+  playbooks_dir = 'ansible/'
+  target = get_target(host)
+  target.run("mkdir -p /srv/playbooks")
+  dest = "/srv/playbooks/"
```

Comment:
Either we define all variables in the first lines, either we define it before using it :) But not randomly spread haha

Reply:
Yeah, this is still WIP so it has not yet the better looking :wink: